_data/fellows.yml

```diff
@@ -1,15 +1,102 @@
 - name: Will
   img: logo.jpg
-  location: London, UK
+  git: https://github.com/wrussell1999
+  location:
+    name: London, UK
+    longitude: -0.118092
+    latitude: 51.509865
 
-- name: Amanda
-  img: amanda.jpg
-  location: New York City, USA
+- name: Grace Omotoso
+  git: https://github.com/grace-omotoso
+  location:
+    name: Murfreesboro, United States
+    latitude: 35.846111
+    longitude: -86.391945
 
-- name: Aaron
-  img: aaron.jpg
-  location: London, UK
+- name: Sai Keshari
+  git: https://github.com/saikeshari
+  location:
+    name: Delhi, India
+    latitude: 28.556160
+    longitude: 77.100281
 
-- name: Swift
-  img: swift.jpg
-  location: New York City, USA
+- name: Neha Kumari
+  git: https://github.com/neha-93
+  location:
+    name: Ranchi, India
+    longitude: 85.296013
+    latitude: 23.344315
+
+
+- name: Yuheng Wu
+  git: https://github.com/iamwyh2019
+  location:
+    name: Beijing, China
+    latitude: 39.916668
+    longitude: 116.383331
+
+- name: Abdulrahman Yusuf
+  git: https://github.com/Young-Einstein10
+  location:
+    name: Lagos, Nigeria
+    longitude: 3.406448
+    latitude: 6.465422
+
+
+- name: Ojuswi Rastogi
+  git: https://github.com/OjusWiZard
+  location:
+    name: Noida, India
+    longitude: 77.391029
+    latitude: 28.535517
+
+
+- name: Gaurav Kumar
+  git: https://github.com/GrayFlash
+  location:
+    name: Bharuch, India
+    latitude: 21.705723
+    longitude: 72.998199
+
+- name: Adarsh Naidu
+  git: https://github.com/adarshnaidu
+  location:
+    name: Bengaluru, India
+    longitude: 77.580643
+    latitude: 12.972442
+
+- name: Priya Bihani
+  git: https://github.com/PriyaBihani
+  location:
+    name: Delhi, India
+    latitude: 28.6691
+    longitude: 77.0929
+
+
+- name: Akshita Dixit
```

Comment:
No need to display this on the page. Just seeing the fellows on the map is fine

Reply:
Hey, even we feel that the way I made changes to fellows.yml file has resulted in a lot of issues in part of page, where we are not making changes, but the intent behind adding things to `fellow.yml` was to avoid redundancy of storing the data around fellows in different files. I think it would have been better if we could have completed `fellow.yml` with all the required details together on the day 0 itself, and later worked on our issues to utilize those data necessary for our issues from there. wdyt??

Anyways, I think I should change the name of the file containing map locations and restore the `fellows.yml` that was present originally. This will help us avoid interference with other teams commits. Makes sense??? @grace-omotoso @OjusWiZard

Reply:
Yep, waiting for the PR updates. Just add the map, don't break things,  lol